components/tabs/demo/tabs.html

```diff
@@ -118,6 +118,27 @@ <h3>Tabs (max-to-show)</h3>
 				</template>
 			</d2l-demo-snippet>
 
+			<h2>Tabs (nested)</h2>
+
+			<d2l-demo-snippet>
+				<template>
+					<d2l-tabs>
+						<d2l-tab-panel text="Biology">Tab content for Biology</d2l-tab-panel>
+						<d2l-tab-panel text="Earth Sciences">
+							Nested tabs for Earth Sciences...
+							<d2l-tabs>
+								<d2l-tab-panel text="Mineralogy">Tab content for Mineralogy</d2l-tab-panel>
+								<d2l-tab-panel text="Hydrology">Tab content for Hydrology</d2l-tab-panel>
+								<d2l-tab-panel text="Geochemistry">Tab content for Geochemistry</d2l-tab-panel>
+								<d2l-tab-panel text="Groundwater Modeling">Tab content for Groundwater Modeling</d2l-tab-panel>
+								<d2l-tab-panel text="Stratigraphy">Tab content for Stratigraphy</d2l-tab-panel>
+							</d2l-tabs>
+						</d2l-tab-panel>
+						<d2l-tab-panel text="Physics">Tab content for Physics</d2l-tab-panel>
+					</d2l-tabs>
+				</template>
+			</d2l-demo-snippet>
+
```

Comment:
Personally I don't like it when really weird edge cases that we don't expect to see often "in the wild" take up space in our demo pages -- which are meant to be a showcase of the component.

How hard would it be to move this into a test that exercises the error and now the solution?

Reply:
I'm am tempted to just ignore it in that case.

Reply:
Yeah I could write one in my-courses, but I'm also working to get rid of the nesting there since it's not necessary.  And then I don't know of anywhere else that uses it... maybe if someone has put a filter dropdown inside a tab panel somewhere?